test/integration/deployment/util.go

```diff
@@ -375,3 +406,24 @@ func (d *deploymentTester) listUpdatedPods() ([]v1.Pod, error) {
 	}
 	return ownedPods, nil
 }
+
+func (d *deploymentTester) waitForReplicaSetHasDesiredReplicas(replicaset *v1beta1.ReplicaSet) error {
```

Comment:
This is a replicaset util, not a deployment util. This can probably be combined with `waitRSStable()` there.

Reply:
How about putting it here: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/replicaset/replica_set_utils.go

Reply:
@janetkuo How about putting it [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/replicaset/replica_set_utils.go)?